src/styles/blocks/card.scss

```
@@ -60,6 +71,7 @@
     color: $color-white;
     background-color: $color-blue;
     border-radius: 5px;
+    transition: color, background-color $animation-time ease-out;
```

This will work even smoothly

```suggestion
    transition: 
      color $animation-time ease-out, 
      background-color $animation-time ease-out;
```